dataactbroker/handlers/fileHandler.py

```diff
@@ -702,7 +699,7 @@ def handleEmptyResponse(self, job, valJob):
             valJob.number_of_errors = 0
             valJob.number_of_warnings = 0
             valJob.filename = None
-            jobDb.session.commit()
+            sess.commit()
```

Comment:
Do we need a `sess.commit()` both here and on line 695? Can we remove the first one? Do we need it on line 687 either? Can we just `sess.commit()` at the very end of this function outside the `if`?